src/ring_theory/power_series.lean

```diff
@@ -164,14 +164,14 @@ end
 protected lemma mul_one : φ * 1 = φ :=
 ext $ λ n,
 begin
-  rw [coeff_mul, finset.sum_eq_single (n, (0 : σ →₀ ℕ))];
-  simp [mem_antidiagonal_support, coeff_one],
```

Comment:
This is a good example of why nonterminal `simp`s are bad. This `simp` was killing two of three goals, and making progress on the third.